### ISSUE TITLE
Add wrap for koral:distance and cosmas:distance

### ISSUE THREAD

**Issue by margaretha** [KorAP/Koral#95]
For coherence, `wrap` should be added in `koral:distance` and `cosmas:distance` for specifying the scope of the distance, e.g. sentence.
<pre>
{
  "@type" : "koral:distance",
  "wrap" : {
    "@type" : "koral:term",
    "foundry" : "base",
    "layer" : "s",
    "key" : "s"
  }
  "boundary" : {...}
}
</pre>

For token distance such as "w" in C2, the `key` attribute should be replaced with "koral:token" as follows:

<pre>
{
  "@type" : "cosmas:distance",
  "wrap" : {
    "@type" : "koral:token"
  }
  "boundary" : {...}
}
</pre>

**Comment by Akron** [comment]
Maybe it's more appropriate to have an "operand" key that wraps a span or a token, that then wraps a term (as usual)?

**Comment by margaretha** [comment]
Yes sure, so for clarification, the serialization should be as follows.

**Sentence**
<pre>
{
    "@type": "koral:distance",
    "key": {
        "@type": "koral:span",
        "wrap": {
            "@type": "koral:term",
            "foundry": "base",
            "layer": "s",
            "key": "s"
        }
    },
    "boundary": {...}
}
</pre>

**Term**
<pre>
{
    "@type": "cosmas:distance",
    "key": {
        "@type": "koral:token"
    },
    "boundary": {...}
}</pre>

**Comment by Akron** [comment]
I literally meant `operand`, because `operand` and `wrap` are the keys to follow for the rewrite, right?
And regarding distances: I guess we agree that exclusion is different to "distance", so we may want to think about renaming `distances` as a key to `constraints` and introduce `koral:exclusion` and `cosmas:exclusion`?
